docs/testnet-rewards.md

```diff
@@ -1,13 +1,13 @@
-# DEVxDAO Casper Testnet Rewards
+# Casper Testnet Rewards
 
 ## Introduction
 
-Thank you for participating in the DEVxDAO's Casper Testnet program. Your voluntary participation in this Testnet is enabling 
+Thank you for participating in the Casper Testnet program. Your voluntary participation in this Testnet is enabling 
```

Comment:
Let's update these references not just here, but also in the footer that's appended to the Testnet related pages

Reply:
Done.